planetiler-custommap/src/main/resources/samples/shortbread.spec.yml

```diff
@@ -39,7 +39,7 @@ examples:
 - name: waterway=dock
   input:
     source: osm
-    geometry: polygon
+    geometry: POLYGON ((0 0, 10 0, 10 10, 0 10, 0 0))
```

Comment:
Accoridng to my logic, this would be a 100m2 area.
Planetiler somewhat disagrees, no idea why

![image](https://github.com/user-attachments/assets/04ca35e2-908c-41de-8ac6-f52657b052e6)
![image](https://github.com/user-attachments/assets/d408784f-bb3d-4c80-8de5-71885f61bdc8)

Reply:
Also unsure in which coordinate system the input is, it is likely not this one
![image](https://github.com/user-attachments/assets/85996622-453e-4332-9609-fab7dc7a5e45)

Reply:
the WKT geometries should be using lon, lat coordinates. 1.23e12 square meters is about 0.2% of earth's surface which sounds about right for 10x10 degrees at the equator.